src/ui/context/SourceContext.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const SourceContext = React.createContext({});
+
+export function SourceContextProvider(props) {
+  const { children } = props;
+  const [show, setShow] = React.useState(true);
```

Comment:
I feel like we should either default to hide the source or default to show it horizontally. With larger, full page components having the source always defaulting to vertical and open might get annoying

Reply:
That works! will make that change to show horizontally, I feel like showing source code by default helps devs know that that feature is available